src/components/Form/FormProvider.js

```diff
@@ -208,6 +215,8 @@ function FormProvider({validate, formID, shouldValidateOnBlur, shouldValidateOnC
 
             if (!_.isUndefined(propsToParse.value)) {
                 inputValues[inputID] = propsToParse.value;
+            } else if(propsToParse.shouldSaveDraft && draftValues[inputID] !== undefined && inputValues[inputID] === undefined) {
+                inputValues[inputID] = draftValues[inputID];
```

Comment:
What is the use for the last condition `inputValues[inputID] === undefined`? Is there a case where `inputValues[inputID]` would have a value when calling `registerInput`?

Reply:
Yes, the function runs each time the input is re-rendered, so it's likely going to happen that it will be called after the user inputs something.

Reply:
I see but what about `shouldUseDefaultValue`. We don't use that condition there